README.md

```diff
@@ -180,6 +180,25 @@ automatic migrations by adding this line to the `Procfile`:
 Note that a failing release will prevent the rolling update from happening, so
 you have to keep compatibility with old code.
 
+### CronJob
+
+**Q: How to create a CronJob?**
+
+    $ teresa app create <app-name> --team <team-name> --process-type cron
+
+Make sure to have the `cron` key with the cronjob command.
```

Comment:
Maybe
```
Make sure to have the `cron` entry in procfile with the cronjob command.
```

Reply:
I followed the previous example [114] _You also have to adjust the `Procfile` to have a corresponding `worker` key._ , but, lgtm to change

Reply:
yes, keep the pattern sounds better